packages/editor-ui/src/components/FixedCollectionParameter.vue

```diff
@@ -287,6 +287,13 @@ export default mixins(genericHelpers)
 </script>
 
 <style scoped lang="scss">
+::v-deep {
+	.button {
+		--button-background-color: var(--color-background-base);
+		--button-border-color: var(--color-foreground-base);
+	}
+}
```

Comment:
we should try to avoid adding overrides as much as possible. why are these needed when it was not needed before? can we not add a variation of this theme that works? (applies to the two other overrides in this PR)
if current impl requires this, talk to Max, he can recommend a way to use button without overrides.

Reply:
Unfortunately, we've got a special case here where the tertiary button must have a different background. I've discussed it with Max and there's a planned replacement for the way we visualize the fields in NDV and we're ok to compromise for now.